examples/common/common.yaml

```diff
@@ -70,6 +70,7 @@ spec:
                 - RoundRobin
                 - WeightedLeastRequest
                 - Random
+                - cookie
```

Comment:
Should we make the casing the same for these options? Either PascalCase or something?

Reply:
That’s a good question. I thought about it some when I chose `cookie`, IMO the pascal case Of the other options are weird because they are a protrusion of envoy’s naming scheme into our world. Lower case cookie was me sort of saying that this isn’t an envoy assigned strategy, but one we synthesise ourselves. 

What I think we should do in the future is relax the marching on that field, if possible, and match on the word reguardless of case. But having said all that, I’ll make it `Cookie` to be consistent with the other values.

Reply:
I agree I don't like the pascal case all that much. =)

Reply:
Yeah, it just feels out of place in k8s, the predominant style is either a pithy lower case word or camelCase. I’m not sure if calling it `cookie` struck a blow for good, or just chaos